templates/es6/www

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env node
+
+/**
+ * Entrypoint to application for es6 support
+ */
+ 
+require("babel-core/register");
+
+/**
+ * Module dependencies.
+ */
+
+var app = require('../app').default;
```

Comment:
Nothing in bin/www can be in ES6 format. The call to babel-core/register patches the require statements and any code executing other code from this file (ala ../app) will run in full ES6 compatility